### PR TITLE
Setup Payload CMS as the data source

### PR DESCRIPTION
**Pull request by heypoom**
Initializing the Payload CMS with `pnpx create-payload-app` according to @bombnp's suggestions.

### CODE REVIEW

**Comment by bombnp** [comment]
That's crazy fast lol. Will review after dinner kub.

File: cms/.gitignore [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+# Ignore all local history of files
+.history
+.ionide
+
+# Support for Project snippet scope
+.vscode/*.code-snippets
+
+# Ignore code-workspaces
+*.code-workspace
+
```
Comment (bombnp) [inline cms/.gitignore:165]: add `build/`, this folder is created during `pnpm build`

**Review by bombnp — CHANGES_REQUESTED** [review]

**Comment by bombnp** [comment]
The `Dockerfile` generated from create-payload-app is horrible lmao. I'll make a new one that uses pnpm. Can I push to this branch directly?

**Comment by heypoom** [comment]
yep feel free to @bombnp

File: cms/.prettierrc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "semi": false,
+  "arrowParens": "avoid",
+  "singleQuote": true,
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
+  "trailingComma": "es5",
+  "bracketSpacing": true,
+  "endOfLine": "lf",
+  "plugins": ["prettier-plugin-astro"],
```
[the remark this answers is not in the document]
Comment (heypoom) [reply cms/.prettierrc:11]: good catch - copied that from the parent .prettierrc. will fix

**Review by rayriffy — APPROVED** [review]
i think at this point maybe we have to move sites into pnpm workspaces. but this could be done in future PR

**Review by bombnp — APPROVED** [review]
There are potential improvements to `payload.config.ts`, but let's leave it to next PRs.

Looks good to me!

**Comment by heypoom** [comment]
okie merged - thanks @bombnp @rayriffy!

**Pull request merged**